hazelcast/src/main/java/com/hazelcast/monitor/TimedMemberState.java

```diff
@@ -155,6 +169,9 @@ public void fromJson(JsonObject json) {
         memberState = new MemberStateImpl();
         memberState.fromJson(jsonMemberState);
         sslEnabled = getBoolean(json, "sslEnabled", false);
+        if (memberState.getNodeState().getClusterVersion().isGreaterOrEqual(Versions.V3_9)) {
```

Comment:
@mesutcelik Thinking about this again, I realized that a member with codebase version 3.9 but cluster version 3.8 cannot use 3.9 features, so it makes sense to only send this piece of data only if the cluster version is 3.9.